src/audio/detect_test.c

```diff
@@ -54,7 +54,8 @@
 	((int16_t)((INT16_MAX) * (ACTIVATION_DEFAULT_THRESHOLD)))
 
 /* number of samples to be treated as a full keyphrase */
-#define KEYPHRASE_DEFAULT_PREAMBLE_LENGTH (30 * 1024)
+#define KEYPHRASE_DEFAULT_PREAMBLE_LENGTH \
+	(KPB_MAX_BUFF_TIME / KPB_SAMPLING_WIDTH)
```

Comment:
Here you are adding more and more dependencies to kpb? @slawblauciak what do you think about this?

Reply:
We could discuss if it is good or bad however I decided to set this default preamble to 0 as this seems to be more natural. By default you expect KWD to be active from start, right?